lib/utils/index.ts

```diff
@@ -47,15 +45,15 @@ export const getCleanType = (abiType: string, toLower = true) => {
 };
 
 export const getJSType = (abiType: string): string => {
-  const cleanType = getCleanType(abiType);
+  const cleanType = getCleanType(abiType, true);
```

Comment:
If the entire `ABITypeMap` file is in lowerCase, in which scenario the toLower will be false? it is really needed in the cleanType method?